tgui/packages/tgui/interfaces/Terminal.js

```diff
@@ -0,0 +1,18 @@
+import { useBackend } from '../backend';
+import { NoticeBox } from '../components';
+import { Window } from '../layouts';
+import { sanitizeText } from '../sanitize';
+
+export const Terminal = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window theme={data.tguitheme} title="Terminal" width={600} height={600}>
+      <Window.Content scrollable>
+        <NoticeBox textAlign="left">
+          {sanitizeText(data.uppertext)}
+        </NoticeBox>
+        {sanitizeText(data.text)}
```

Comment:
This is why you're getting `&lt;` etc. Why are you using this? tgui automatically sanitizes, you only need this if you're pasting in content raw, like through unsafeInnerHTML or direct DOM calls.